frontend/sentiment-dashboard/src/app/auth/auth.service.ts

```diff
@@ -56,4 +56,21 @@ export class AuthService {
       })
     );
   }
+
+  public isAdmin(): Observable<boolean> {
+    return from(Auth.currentSession()).pipe(
+      map(session => {
+        const userGroups = session.getIdToken().decodePayload()['cognito:groups'];
+        if (userGroups !== undefined) {
+          const found = userGroups.includes('admin');
+          if (userGroups.includes('admin')) {
```

Comment:
Should this be `if (found)`? It might be more concise just to `return found` here.